src/libraries/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.cs

```diff
@@ -25,6 +25,10 @@ private static partial class Settings
         // This allows for the whole Settings nested class to be trimmed when Invariant=true, and allows for the Settings
         // static cctor (on Unix) to be preserved when Invariant=false.
         internal static bool Invariant => Settings.Invariant;
+
```

Comment:
What about putting it into `#if TARGET_BROWSER`? Same about changes in `CultureData`